models/jira__issue_enhanced.sql

```diff
@@ -18,7 +36,12 @@ final as (
         -- if an issue is not currently assigned this will not be null
         {{ dbt_utils.datediff('last_assigned_at', "coalesce(resolved_at, " ~ dbt_utils.current_timestamp() ~ ')', 'second') }} last_assignment_duration_seconds 
     
+        {% for col in pivot_data_columns if col.name|lower not in ['issue_day_id','issue_id','latest_record', 'date_day'] %} 
+        , {{ col.name }}
+        {% endfor %}
+
     from issue
+    left join latest_issue_field_history using (issue_id)
```

Comment:
fyi, we had some weird issues with `using` when migrating from postgres to snowflake so we've taken the approach of always joining with `on` clause

Reply:
good to know! I've just updated to use `on`. Will share this feedback with my team as well.

Reply:
thanks. For reference, we hit an issue similar to what's [described here](https://community.snowflake.com/s/question/0D50Z00008WRZBBSA5/bug-with-join-using-) (though not only on full outer joins) and the response from Snowflake:
> Just want to clarify that this is not a bug but a different semantics because this USING syntax is not actually in the ANSI standard and we don't encourage user to use which is why we don't document it. To support it is just for compatibility ie for migration from another data source into snowflake